.github/workflows/test.yaml

```diff
@@ -128,6 +128,8 @@ jobs:
                 dependency-versions: ${{ matrix.dependency-version }}
 
             - name: ${{ matrix.component }} Tests
+              env:
+                  SYMFONY_DEPRECATIONS_HELPER: 'max[total]=999999'
```

Comment:
Could you create a separate PR for this change ?

Reply:
Sure thing, the question is whether we want (or must) do it this way. It's a little hack, but on the other hand, the question is whether we need this information in the CI.